frontend/src/v4/modules/sequences/sequences.sagas.ts

```diff
@@ -114,76 +106,34 @@ export function* fetchFrame({ date }) {
 		const teamspace = yield select(selectCurrentModelTeamspace);
 		const model = yield select(selectSequenceModel);
 		const sequenceId =  yield select(selectSelectedSequenceId);
-		const loadedStates = yield select(selectStateDefinitions);
 		const frames = yield select(selectFrames);
-		const selectedDate = yield select(selectSelectedDate);
-		const dateIsSelectedDate = selectedDate.valueOf() === date.valueOf()
-		const { state: stateId, viewpoint } = getSelectedFrame(frames, date);
+		const { state: stateId } = getSelectedFrame(frames, date);
 
 		if (stateId) {
 			const cacheEnabled = yield select(selectCacheSetting);
-			const iDBKey = `${teamspace}.${model}.${stateId}.3DRepo`;
-
-			const cachedDataPromise = cacheEnabled ? DataCache.getValue(STORE_NAME.FRAMES, iDBKey) : Promise.resolve();
-
-			if (!loadedStates[stateId]) {
-				// Using directly the promise and 'then' to dispatch the rest of the actions
-				// because with yield it would sometimes stop there forever even though the promise resolved
-				cachedDataPromise.then((cachedData) => {
-					const fetchPromise = cachedData ? Promise.resolve({data: cachedData})
-						: API.getSequenceState(teamspace, model, sequenceId, stateId);
-					fetchPromise.then((response) => {
-						dispatch(SequencesActions.setStateDefinition(stateId, response.data));
-						if (dateIsSelectedDate) {
-							dispatch(SequencesActions.setLastSelectedDateSuccess(date));
-							dispatch(SequencesActions.updateSelectedStateDefinition());
-						}
-						if (cacheEnabled && !cachedData) {
-							DataCache.putValue(STORE_NAME.FRAMES, iDBKey, response.data);
-						}
-					});
-				}).catch((e) => {
-					dispatch(SequencesActions.setStateDefinition(stateId, {}));
-				});
+			const iDBKey = `${teamspace}.${model}.${stateId}.Asite3DRepo`;
```

Comment:
Changed the key here because I changed from caching the state to caching the viewpoint.